bigquery_etl/cli/dryrun.py

```diff
@@ -27,8 +30,8 @@
     """,
 )
 @click.argument(
-    "path",
-    default="sql/",
+    "paths",
+    nargs=-1,
```

Comment:
replaced `path` with `paths` so that `bqetl dryrun` could replace `python3 -m bigquery_etl.dryrun`